content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -80,6 +89,8 @@ export class _StartupOverlay extends React.PureComponent {
                 <input name="entrypoint" type="hidden" value="activity-stream-firstrun" />
```

Comment:
@ericawright this form above has `accounts.firefox.com` hardcoded vs using `this.props.fxa_metrics_endpoint` like the fetch script. Is it possible to fix that?

Reply:
yeah I think that's already in the prefs, I'll find it